src/main/resources/messages.properties

```diff
@@ -10,12 +10,15 @@ invalid.username=Please enter valid login, do not use space
 invalid.past-or-present.date=${validatedValue} date couldn't be in the future
 invalid.password=Your password must contain at least: 1 uppercase Latin character (A-Z), 1 lowercase Latin character (a-z), 1 digit (0-9)
 invalid.password.size=Your password must be between 8 and 30 characters long
+incorrect.password=Please enter valid password
 invalid.username.size=Your login must be between 2 and 50 characters, do not use space
 invalid.username-or-password=Please enter valid email/login or password
 invalid.phone.number=Phone number has to contain your country code and no letters (Ex. +380501234567 or +1(234)567-89-10)
 invalid.first-or-last.name=${validatedValue} must be between 2 and 50 characters (allowed only ['] and [-] symbols)
+invalid.confirm.email=Invalid confirmation of email. Make sure the new email matches the confirmation of email
 username.duplicate=This login already exists. Please, come up with another login
 email.duplicate=This email already exists. Please, enter another email or sign in
+email.old=You have entered your old email address. If you would like to change your email address, please try again
```

Comment:
As I see you're using this text into an exception, so you need to put prefix "exception" here or "invalid".

Reply:
done